src/os_push_notifications.eliomi

```diff
@@ -123,29 +124,29 @@ module Notification :
       end
 
     (** Add two buttons with an action (created with Action.create). Be sure you
-     * exported the callback in JavaScript.
+        exported the callback in JavaScript.
      *)
     val add_actions : Action.t -> Action.t -> t -> t
 
     (** Change the led color when the notification is received. The parameters
-     * are in the ARGB format.
+        are in the ARGB format.
      *)
     val add_led_color : int -> int -> int -> int -> t -> t
 
     (** Add a vibration pattern *)
     val add_vibration_pattern : int list -> t -> t
 
     (** Add a badge to the icon of the notification in the launcher. Only
-     * available for some launcher. The integer parameter is the number of the
-     * badge.
+        available for some launcher. The integer parameter is the number of the
+        badge.
      *)
     val add_badge : int -> t -> t
 
     module Priority :
       sig
-        (* Maximum means the notification will be displayed on the screen above
-         * all views during 2 or 3 seconds. The notification still available in
-         * the notification area.
+        (** Maximum means the notification will be displayed on the screen above
+            all views during 2 or 3 seconds. The notification still available in
+            the notification area.
```

Comment:
The notification still available in the notification area. ===> The notification will still be available in the notification area. ? The notification remains available in the notification area. ?